project.clj

```diff
@@ -7,30 +7,28 @@
                         ["snapshots" :clojars]]
   :plugins [[lein-ancient "0.7.0"]
             [lein-nvd "1.4.1"]]
-  :dependencies [[org.clojure/clojure "1.10.3"]
-                 [org.clojure/core.async "1.3.610" :exclusions [org.clojure/tools.reader]]
-                 [prismatic/schema "1.1.12"]
+  :dependencies [[org.clojure/clojure "1.11.1"]
+                 [org.clojure/core.async "1.5.648"]
+                 [prismatic/schema "1.2.0"]
                  [camel-snake-kebab "0.4.2"]
-                 [com.cognitect/transit-clj "1.0.324"]
-                 [com.cognitect/transit-cljs "0.8.264"]
+                 [com.cognitect/transit-clj "1.0.329"]
+                 [com.cognitect/transit-cljs "0.8.269"]
                  [commons-codec/commons-codec "1.15"]
-                 [metosin/ring-http-response "0.9.2" :exclusions [ring/ring-core]]
-                 [org.apache.commons/commons-csv "1.8"]
-                 [commons-io/commons-io "2.8.0"]
-                 [info.sunng/ring-jetty9-adapter "0.14.3" :exclusions [org.eclipse.jetty/jetty-alpn-conscrypt-server
-                                                                       org.conscrypt/conscrypt-openjdk-uber
-                                                                       org.eclipse.jetty/jetty-server]]
+                 [metosin/ring-http-response "0.9.3" :exclusions [ring/ring-core]]
+                 [org.apache.commons/commons-csv "1.9.0"]
+                 [commons-io/commons-io "2.11.0"]
+                 [info.sunng/ring-jetty9-adapter "0.17.6" :exclusions [org.eclipse.jetty/jetty-server]]
                  ;; cf. https://github.com/advisories/GHSA-vjv5-gp2w-65vm
-                 [org.eclipse.jetty/jetty-server "9.4.44.v20210927"]
-                 [com.stuartsierra/component "1.0.0"]
-                 [hikari-cp "2.13.0"]
-                 [com.github.seancorfield/next.jdbc "1.1.646"]
-                 [metosin/jsonista "0.3.1"]
+                 [org.eclipse.jetty/jetty-server "10.0.9"]
```

Comment:
[`ring-jetty9-adapter` depends on jetty 10.0.9](https://github.com/sunng87/ring-jetty9-adapter/blob/master/project.clj#L1), so this line is redundant.

And Jetty 10 drops Java8 so I want you to write it on change log .